encode/hevcencode.c

```diff
@@ -1808,7 +1812,7 @@ static int process_cmdline(int argc, char *argv[])
             print_help();
             exit(0);
         case 1:
-            frame_bitrate = atoi(optarg);
+            frame_bitrate = atoi(optarg)*1000;
```

Comment:
so, it was changed to K, could you help to add the description in command help

Reply:
Line 1571, I add info to indicate Kbits.
printf("   --bitrate <bitrate> Kbits per second\n");
Where else did I forget to add the description?